index.html

```diff
@@ -33,7 +33,7 @@
       The <a href="licenses/apache-2.0/">Apache License 2.0</a> is a permissive license similar to the MIT License, but also provides an express grant of patent rights from contributors to users.
     </p>
     <p>
-      <strong>Apache</strong>, <strong>Swift</strong>, and <strong>Android</strong> use the Apache&nbsp;License&nbsp;2.0.
+      <a href="https://httpd.apache.org/">Apache</a>, <a href="https://github.com/apple/swift/blob/master/LICENSE.txt">Swift</a>, and <a href="https://github.com/android/platform_system_core/blob/master/NOTICE">Android</a> use the Apache&nbsp;License&nbsp;2.0.
```

Comment:
The LICENSE file in the Apache source seems to be [here][1].

[1]: http://svn.apache.org/viewvc/httpd/httpd/trunk/LICENSE?view=markup